app/lib/openingTimesMapper.js

```diff
@@ -65,10 +17,10 @@ function mapWeek(times) {
 
   daysOfWeek.forEach((day) => {
     const daySessions = times[day.toLowerCase()];
-    const sessions = mapDay(daySessions);
+    const sessions = continuousTimeUtils.mapKey(daySessions);
```

Comment:
I would have left this function named 'mapDay' when moving it, as both an alteration date and a day of the week represent times for a day. 
`mapKey` is a bit generic.